storage/innobase/handler/handler0alter.cc

```diff
@@ -1489,6 +1499,10 @@ instant_alter_column_possible(
 	ut_ad(pk->is_primary());
 	ut_ad(!pk->has_virtual());
 
+	if (ha_alter_info->handler_flags & ALTER_PERSISTENT_COUNT_ONOFF) {
+		return true;
+	}
```

Comment:
I don’t think that enabling a persistent count is an instantaneous operation, because it will involve scanning the entire table, to count the records.

Reply:
Ah, I had added this conditional return so that `ALTER PERSISTENT_COUNT` would take the same execution path as `INSTANT DROP COLUMN`. Will fix.